robosuite/environments/base.py

```diff
@@ -516,7 +512,7 @@ def render(self):
         """
         Renders to an on-screen window.
         """
-        self.viewer.render()
+        return self.viewer.render()
```

Comment:
this function is supposed to perform onscreen rendering rather than return any values